src/levels/1/1-1/1-1-area0.asm

```diff
@@ -1,7 +1,8 @@
 ; Level 1-1, Area 0
 
 LevelData_1_1_Area0:
-	.db $09, $F3, $30, $00
+	levelHeader 3, LevelDirection_Vertical, 1, 1, LevelMusic_Overworld, 0, 0, $13, $0
```

Comment:
The only issue I have with this macro is that it's largely incomprehensible without knowing the param list. For something short like `enemy x, y, type` it isn't _quite_ bad, but for this, there's a lot of really similar information.

Unfortunately, the only likely solution to this is including a comment above it, like
`; Pages, Orientation, Ground type [0-7], ground setting [0-31], Music, ?, ?, ?, ?` (etc)

that adds a lot of redundancy, but at least it makes it a little more readable, and it's an easy thing to search-and-replace.